### PR TITLE
Release for 2.2

### PR DESCRIPTION
**Pull request by sergiusens**
Signed-off-by: Sergio Schvezov <sergio.schvezov@ubuntu.com>

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,36 @@
...
+  * Add the unit tests to the autopkgtest suite. (LP: #1541421)
+  * Use docopt to handle example tests arguments. (LP: #1533395)
+  * integration-tests: add login and logout tests (LP: #1541391)
+  * move image into docs/images (LP: #1542422)
+  * integration_tests: add a successful upload test (LP: #1541391)
+  * travis: remove the coveralls token (LP: #1543675)
+  * examples_tests: decode the output with utf-8 (LP: #1543714)
+
+  [ Daniel Holbach ]
+  * Clear APT::Update::Post-Invoke-Success so we don't scripts like
```
Comment (kyrofa) [inline debian/changelog]: Worth fixing the "so we don't scripts" typo here?

**Comment by kyrofa** [comment]
Yeah that's a pretty decent-looking release if you ask me!

File: debian/changelog (hunk 1 above)
Comment (sergiusens) [inline debian/changelog]: done

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,36 @@
+snapcraft (2.2) xenial; urgency=medium
+
+  [ Leo Arias ]
+  * Add the unit tests to the autopkgtest suite. (LP: #1541421)
+  * Use docopt to handle example tests arguments. (LP: #1533395)
+  * integration-tests: add login and logout tests (LP: #1541391)
+  * move image into docs/images (LP: #1542422)
```
Comment (elopio) [inline debian/changelog]: I didn't do this one.

Comment (kyrofa) [inline debian/changelog]: Good catch, that was @dholbach.

Comment (sergiusens) [inline debian/changelog]: Anything else? Not sure how this happened; but there were like 3 or 4 @elopio's around, so I might have confused this :wink:

**Pull request merged**